src/components/TransactionConfirmationModal/index.tsx

```diff
@@ -63,15 +64,35 @@ export function ConfirmationPendingContent({ onDismiss, pendingText }: { onDismi
 }
 
 function TransactionSubmittedContent({
+  trade,
   onDismiss,
   chainId,
   hash,
 }: {
+  trade?: Trade
   onDismiss: () => void
   hash: string | undefined
   chainId: ChainId
 }) {
   const theme = useTheme()
+  const { t } = useTranslation()
+
+  const isGnosisProtocolTrade = trade instanceof GnosisProtocolTrade
+  const explorer = isGnosisProtocolTrade ? t('gnosisProtocolExplorer') : t('blockExplorer')
+  const href =
```

Comment:
Same as comment above, we should be consistent with naming, use `link` or `href` in all the cases.

Reply:
done.